Makefile.am

```diff
@@ -333,21 +313,27 @@ tests_readclause_test_SOURCES += tests/dummy.cpp
 tests_ssml_fuzzer_test_SOURCES += tests/dummy.cpp
 endif
 
+tests_fuzzing_synth_fuzzer_SOURCES = tests/fuzzing/synth_fuzzer.c
+tests_fuzzing_synth_fuzzer_LDADD =  src/libespeak-ng.la
+tests_fuzzing_synth_fuzzer_CFLAGS=  -fsanitize=address,undefined,fuzzer -fno-omit-frame-pointer -g -fsanitize-coverage=edge -Isrc/libespeak-ng -g -O2 -fdebug-prefix-map=/root=. -fstack-protector-strong -Wformat -Werror=format-security -O1  -g -ggdb3  -fprofile-instr-generate -fcoverage-mapping ${AM_CFLAGS}
+tests_fuzzing_synth_fuzzer_LDFLAGS= -fsanitize=address,undefined,fuzzer -static -lm  -fsanitize-coverage=edge -Wl,-z,relro ${PCAUDIOLIB_LIBS}
+
+tests_ssml_fuzzer_SOURCES = tests/ssml-fuzzer.c
+tests_ssml_fuzzer_LDADD =  src/libespeak-ng.la
+tests_ssml_fuzzer_CFLAGS= -fsanitize=address,undefined,fuzzer -fno-omit-frame-pointer -g -fsanitize-coverage=edge -Isrc/libespeak-ng ${AM_CFLAGS}
+tests_ssml_fuzzer_LDFLAGS= -fsanitize=address,undefined,fuzzer -static -lm  -fsanitize-coverage=edge ${PCAUDIOLIB_LIBS}
+
 if HAVE_LIBFUZZER
-tests_ssml_fuzzer_test_CFLAGS += -fsanitize=fuzzer
-tests_ssml_fuzzer_test_LDFLAGS = -fsanitize=fuzzer
+bin_PROGRAMS +=  tests/fuzzing/synth_fuzzer
+bin_PROGRAMS +=  tests/ssml_fuzzer
```

Comment:
This should rather be `noinst_PROGRAMS`, since these are not tools for end users